documentation/en/setting-a-static-port.md

```diff
@@ -0,0 +1,42 @@
+# Static port
+
+If you plan to accept a **storage deal**, you will want to set a static port and open it in your firewall to ensure clients can connect to you.
+
+Lotus binds to a random **swarm port** by default.
+
+## Setup
+
+To change the random **swarm port**, you may edit the `config.toml` file located under `$LOTUS_PATH`. The default location of this file is `$HOME/.lotus`.
```

Comment:
While this can be done in `.lotus`, it's more important to do that in `.lotusstorage` (miner repo), which has a separate libp2p node, and it's actually the node that receives deals